src/main/resources/assets/tconstruct/book/tinkers_gadgetry/zh_cn/ancient/tconstruct_war_pick.json

```
@@ -0,0 +1,12 @@
+{
+  "tool": "tconstruct:war_pick",
+  "text": [
+    { "text": "战镐是镐和弩的结合体。可以在废弃矿井中、村民或灾厄村民手中获得。" }
```

```suggestion
    { "text": "战镐是镐和弩的结合体。可以从废弃矿井中、村民或灾厄村民手中获得。" }
```
Polishing, not required.